packages/xpath/src/adapter/defaults.ts

```diff
@@ -20,7 +20,7 @@ import { xpathDOMProvider } from './xpathDOMProvider.ts';
  * types to their WHAT* equivalent. This is for clarity at usage and reference
  * sites, internally but especially at the package boundary.
  */
-type AsDefault<T> = T | (T & { readonly _?: never });
+type AsDefault<T> = T | (T & { readonly _?: never }); // eslint-disable-line @typescript-eslint/no-redundant-type-constituents
```

Comment:
This is popping up so frequently maybe we need to disable it in the tsconfig?

Reply:
I've resolved some of these and removed the skip of half of them 🙂